crates/superposition/src/main.rs

```diff
@@ -204,7 +205,13 @@ async fn main() -> Result<()> {
                     )
                     .service(workspace::endpoints(scope("/workspaces"))
                         .wrap(OrgWorkspaceMiddlewareFactory::new(true, false))
-                        .wrap(AppExecutionScopeMiddlewareFactory::new(AppScope::SUPERPOSITION)),
+                        .wrap(AppExecutionScopeMiddlewareFactory::new(AppScope::SUPERPOSITION))
+                    )
+                    .service(
+                        scope("/webhook")
+                            .wrap(OrgWorkspaceMiddlewareFactory::new(true, false))
```

Comment:
shouldn't it be this ?
```suggestion
                            .wrap(OrgWorkspaceMiddlewareFactory::new(true, true))
```